examples/typecheck_examples.sh

```diff
@@ -35,3 +35,6 @@ typecheck hacspec-sha3              no-ec      fst
 typecheck hacspec-gimli                ec      fst
 typecheck hacspec-bls12-381         no-ec   no-fst
 typecheck hacspec-ecdsa-p256-sha256 no-ec   no-fst
+typecheck hacspec-aes               no-ec      fst
```

Comment:
I would recommend putting no-fst by default unless we've checked that the generated F* typechecks out of the box.

Reply:
Ok I see. Adding F* as a dependency to a CI configuration is a small feat in itself, I'll see what I can do.